sections/GPU.tex

```diff
@@ -1,14 +1,21 @@
-\section{Graphics Processing Units}
-\label{sec:GPU}
-Dedicated Graphic Processing Units - GPUs - are very common in desktop and laptops. \citep{STEAMHW}
+\section{Computation architectures}
+\label{sec:comparch}
+Normally computations are performed on a computer's Central Processing Unit - a CPU.
+A CPU is often fastest for everyday task and calculations.
+The CPU consist of only a few cores which are highly optimized for sequential serial processing.\citep{whatisgpu}
+But in recent years developers and scientists seem to have developed at interest for performing calculations on Dedicated Graphic Processing Units.\citep{gpurise}
+This is a whole other architecture which normally only was used for graphics processing but with some problems it seems that one could utilise a GPU for General Purpose GPU computing - GPGPU, with an performance improvement over a CPU.
+In the following text we explore the possibilities that a GPU architecture offers.
+
+GPUs are very common in desktop and laptops. \citep{STEAMHW}
 GPUs are commonly used in the private and professional world, for computer gaming and accelerating graphic intensive programs such as Adobe Photoshop and 3D modelling tools. \citep{NVIDIAADOBE}
 Their architecture can also be used for advantageously for certain types of computations, primarily computations which can be done in parallel. 
 An example of this can be calculating different properties of the pixels on a screen. 
-The screen can then be divided into different sections and the computations for each section is divided over the GPU.
-This is possible because the calculations do not require results from the other calculations.
+The screen can then be divided into different sections and the computations for each section is divided over the GPU internally.
```

Comment:
Sætningen virker lidt underligt formuleret,
[...] different sections called blocks and the computations for each block of data is divided internally on the GPU to its threads